src/viam/ur/module/ur_arm.cpp

```diff
@@ -49,6 +49,17 @@ pose ur_vector_to_pose(urcl::vector6d_t vec) {
     return pose{position, orientation, theta};
 }
 
+// https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/bff7bf2e2a85c17fa3f88adda241763040596ff1/include/ur_client_library/ur/datatypes.h#L204
+std::string to_robot_type_string(const std::string& model) {
+    if (model == "viam:universal-robots:ur5e") {
+        return "UR5";
+    }
+    if (model == "viam:universal-robots:ur20e") {
```

Comment:
there's probably a better way for us to carry the `model_` info, but when combined with the limited return types offered by `commandGetRobotModel` this seemed the most straightforward approach